talend-component-maven-plugin/src/it/web/.jenkins/Jenkinsfile

```diff
@@ -0,0 +1,254 @@
+                { name: dockercache, mountPath: /root/.dockercache }
+              ]
+              resources: { requests: { memory: 3G, cpu: '2' }, limits: { memory: 8G, cpu: '2' } }
+          volumes:
+            - name: docker
+              hostPath: { path: /var/run/docker.sock }
+            - name: efs-jenkins-connectors-se-m2
+              persistentVolumeClaim:
+                    claimName: efs-jenkins-connectors-se-m2
+            - name: dockercache
+              hostPath: { path: /tmp/jenkins/tdi/docker }
+          imagePullSecrets:
+            - name: talend-registry
+           """.stripIndent()
+    }
+  }
+
+  environment {
+    MAVEN_SETTINGS = "${WORKSPACE}/.jenkins/settings.xml"
+    DECRYPTER_ARG = "-Dtalend.maven.decrypter.m2.location=${env.WORKSPACE}/.jenkins/"
+    MAVEN_OPTS = [
+            "-Dmaven.artifact.threads=128",
+            "-Dorg.slf4j.simpleLogger.showDateTime=true",
+            "-Dorg.slf4j.simpleLogger.showThreadName=true",
+            "-Dorg.slf4j.simpleLogger.dateTimeFormat=HH:mm:ss",
+            "-Dtalend-image.layersCacheDirectory=/root/.dockercache"
+    ].join(' ')
+
+    APP_ID = '579232'
+    TALEND_REGISTRY = "artifactory.datapwn.com"
+    TESTCONTAINERS_HUB_IMAGE_NAME_PREFIX = "artifactory.datapwn.com/docker-io-remote/"
+
+    ROOT_PATH = "${WORKSPACE}"
+    SCRIPTS_PATH = "${WORKSPACE}/talend-component-maven-plugin/src/it/web/.jenkins/scripts"
+    TESTS_PATH = "${WORKSPACE}/test_scenarios/talend-component-maven-plugin/src/it/web/test"
+    CONNECTORS_PATH = "${WORKSPACE}/connectors"
+    TESTED_CONNECTOR_PATH = "${CONNECTORS_PATH}/azure/azure-dls-gen2"
+
+    API_TESTER_ID = "$apiTesterId"
+  }
+
+  options {
+    buildDiscarder(logRotator(artifactNumToKeepStr: '50',
+                              numToKeepStr: '50'))
+    timeout(time: 30, unit: 'MINUTES')
+    skipStagesAfterUnstable()
+  }
+
+  stages {
+    stage('Test Info') {
+      steps {
+        container(tsbiImage) {
+          script {
+            println("API Tester ID: ${apiTesterId}")
+            println("Runtime version: ${runtimeVersion}")
+            println("Test branch: ${branchToUse}")
+
+            def branchName = "master"
+            if (branchToUse != "master") {
+                branchName = "branch"
+            }
+
+            def runtimeName = "master"
+            if (runtimeVersion != "") {
+                runtimeName = runtimeVersion.replace("-SNAPSHOT", "-S")
+            }
+            //updating displayName
+            currentBuild.displayName = "#" + currentBuild.number + ": " + branchName + "(" + runtimeName + ")"
+          }
+        }
+      }
+    }
+    stage('Test Checkout') {
+      steps {
+        container(tsbiImage) {
+          // Checkout of the connector directory (TODO Will be cleaned when test component will be ready)
+          checkout([
+              $class           : 'GitSCM',
+              branches         : [[name: '*/master']],
```

Comment:
always checking out master?  may change after when test comp will be available, no?

Reply:
Correct, when the mock connector will be developed, this checkout will be remove, hopefully before the merge of this PR